stream-chat-android-offline/src/main/java/io/getstream/chat/android/livedata/repository/RepositoryHelper.kt

```diff
@@ -18,32 +18,35 @@ import kotlinx.coroutines.async
 import kotlinx.coroutines.awaitAll
 import java.util.Date
 
-internal class RepositoryHelper(
-    factory: RepositoryFactory,
+internal class RepositoryHelper private constructor(
+    private val userRepository: UserRepository,
+    private val configsRepository: ChannelConfigRepository,
+    private val channelsRepository: ChannelRepository,
+    private val queryChannelsRepository: QueryChannelsRepository,
+    private val messageRepository: MessageRepository,
+    private val reactionsRepository: ReactionRepository,
+    private val syncStateRepository: SyncStateRepository,
     private val scope: CoroutineScope,
-) {
-    private val userRepository = factory.createUserRepository()
-    private val configsRepository = factory.createChannelConfigRepository()
-    private val channelsRepository = factory.createChannelRepository()
-    private val queryChannelsRepository = factory.createQueryChannelsRepository()
-    private val messageRepository = factory.createMessageRepository()
-    private val reactionsRepository = factory.createReactionRepository()
-    private val syncStateRepository = factory.createSyncStateRepository()
+) : UserRepository by userRepository {
+
+    private val selectUser: suspend (userId: String) -> User = { userId ->
+        selectUser(userId) ?: error("User with the userId: `$userId` has not been found")
```

Comment:
NIT: `requireNotNull(userRepository.selectUser(userId)) { "User with the userId: `$userId` has not been found" }`

Reply:
I'll change in next PR thanks. But this is temporary member. I will remove it when finish the task